setup.py

```diff
@@ -378,7 +378,8 @@ def _classify_installed_files_override(
         print("Copying files from CMake output")
 
         # add lines from the old __init__.py file to the config file
-        with open('scripts/__init__.py', 'r') as custom_init:
+        with open('%s/scripts/__init__.py'
+        % os.path.split(os.path.abspath(__file__))[0], 'r') as custom_init:
```

Comment:
os.path.join should be better than  "%s"

Reply:
Added `os.path.join` usage instead of `%s`.